### PR TITLE
DB-1350: replace "CLIQZ" with "Cliqz"

### PR DESCRIPTION
**Pull request by alver-cliqz**

### CODE REVIEW

File: mozilla-release/browser/installer/windows/nsis/installer.nsi [hunk 1]
```diff
@@ -992,13 +992,13 @@ Function preComponents
 
   ; Only show the maintenance service page if we have write access to HKLM
   ClearErrors
-  WriteRegStr HKLM "Software\CLIQZ" \
+  WriteRegStr HKLM "Software\Cliqz" \
```
Comment (luciancor) [inline mozilla-release/browser/installer/windows/nsis/installer.nsi]: @alver-cliqz what will happen to existing users when we change this pref?

Comment (alver-cliqz) [inline mozilla-release/browser/installer/windows/nsis/installer.nsi]: Nothing. Windows is case insensitive, even with registry. New users will receive new way of writing, for old users it must stay the same. I checked some cases - looks like it work as normal. But we will need to test this more with QA

File: mozilla-release/browser/config/cliqz.mozconfig [hunk 2]
```diff
@@ -7,7 +7,7 @@ ac_add_options --enable-signmar
 
 ac_add_options --with-branding=browser/branding/cliqz
 ac_add_options --enable-official-branding
-ac_add_options --with-app-name=CLIQZ
+ac_add_options --with-app-name=cliqz  # name for binaries
 
-export MOZ_APP_PROFILE=CLIQZ
+export MOZ_APP_PROFILE=Cliqz
```
Comment (chrmod) [inline mozilla-release/browser/config/cliqz.mozconfig]: this probably should be kept CLIQZ all capitals as macos and linux are case sensitive on filesystem names

File: mozilla-release/toolkit/mozapps/extensions/internal/XPIProviderUtils.js [hunk 3]
```diff
@@ -1521,7 +1521,7 @@ this.XPIDatabase = {
     // TODO: Remove that
     setTimeout(function() {
       try {
-        Components.utils.import('chrome://cliqzmodules/content/CLIQZ.jsm')
+        Components.utils.import('chrome://cliqzmodules/content/cliqz.jsm')
```
Comment (chrmod) [inline mozilla-release/toolkit/mozapps/extensions/internal/XPIProviderUtils.js]: this file comes from extension. It should be called `CLIQZ.jsm`

File: repack/distribution/distribution.ini [hunk 4]
```diff
@@ -1,10 +1,10 @@
 # Partner Distribution Configuration File
-# Author: CLIQZ
+# Author: Cliqz
 # Date: 2015-09-29
 
 [Global]
-about.de=CLIQZ
-about=CLIQZ
+about.de=Cliqz
+about=Cliqz
```
Comment (chrmod) [inline repack/distribution/distribution.ini]: where this one is used exactly? Hope it is not sent to balrog or something.

File: repack/repack.cfg [hunk 5]
```diff
@@ -1,5 +1,5 @@
-aus="CLIQZ"
-dist_id="CLIQZ"
+aus="Cliqz"
+dist_id="Cliqz"
```
Comment (chrmod) [inline repack/repack.cfg]: those two definitely goes to balrog. We should not change

File: mozilla-release/toolkit/mozapps/extensions/internal/XPIProviderUtils.js (hunk 3 above)
Comment (alver-cliqz) [inline mozilla-release/toolkit/mozapps/extensions/internal/XPIProviderUtils.js]: my bad, will fix

File: mozilla-release/browser/config/cliqz.mozconfig (hunk 2 above)
Comment (alver-cliqz) [inline mozilla-release/browser/config/cliqz.mozconfig]: yep, you right

**Comment by alver-cliqz** [comment]
@chrmod @luciancor please take a look again

**Comment by alver-cliqz** [comment]
retest this please

**Comment by luciancor** [comment]
@alver-cliqz LGTM

**Pull request merged**